abel/transform.py

```diff
@@ -201,6 +202,13 @@ def transform(
                 is not currently implemented
     """
 
+    abel_transform = {\
```

Comment:
This seems fine. You are just leaving the raising of exceptions to the individual transform functions. Do all of the functions raise an exception if the wrong direction is passed?